business/authentication/openid_auth_controller.go

```diff
@@ -335,6 +306,7 @@ func (c OpenIdAuthController) authenticateWithAuthorizationCodeFlow(r *http.Requ
 			if flow.ShouldTerminateSession {
 				c.SessionStore.TerminateSession(r, w)
 			}
+			log.Warningf("Authentication rejected: %s", flow.Error.Error())
```

Comment:
Maybe this should be a debug log instead of a warning? Users may not want their logs flooded by login requests and I'm not sure what details or credentials the `flow.Error.Error()` message shows.

Reply:
Because of #5638 you no longer will see the details of auth errors in the UI.

I realized that when an error happens, the error does not go to the logs. You also won't see it in the UI. It happened to me that I had to look at the URL to see the `Cannot view any namespaces. Please read Kiali's RBAC documentation for more details.`, which meant that my user didn't have enough privileges to login.

So, I added this to have the error in the logs. I do agree that for some cases it's better a `debug` rather than a `warning`, but there is no way to make distinction here.

Honestly, I think the specific error that made me put this should be shown at the UI, but because of #5638 and it's motivated issue we can't just be passing strings through the URL and showing them at the UI as is. So, we need a way to pass to the UI these kind of errors. That may be as simple as having in the UI a whitelist about what is safe to show... or pass error codes rather than strings.

Reply:
> I realized that when an error happens, the error does not go to the logs. You also won't see it in the UI. It happened to me that I had to look at the URL to see the Cannot view any namespaces. Please read Kiali's RBAC documentation for more details., which meant that my user didn't have enough privileges to login.

I've run into this issue as well. Maybe the frontend can point users to the backend logs for more details? Anyways keeping it as a warning is fine, esp. since the errors from the frontend can be cryptic.

Reply:
I agree! Pointing the user to the logs will give more clarity about where to look.